src/components/Sep6/Sep6Withdraw.tsx

```diff
@@ -219,27 +230,50 @@ export const Sep6Withdraw = () => {
         )}
 
         <div className="ModalButtonsFooter">
-          <Button onClick={handleSubmit}>Submit</Button>
+          <Button onClick={handleFieldsSubmit}>Submit</Button>
         </div>
       </Modal>
     );
   }
 
-  if (sep6WithdrawAsset.status === ActionStatus.SUCCESS) {
+  if (sep6WithdrawAsset.status === ActionStatus.CAN_PROCEED) {
     return (
       <Modal visible={true} onClose={handleClose}>
-        <Heading2 className="ModalHeading">SEP-6 Withdraw Info</Heading2>
+        <Heading2 className="ModalHeading">Payment Sending</Heading2>
 
         <div className="ModalBody">
           <InfoBlock>
```

Comment:
Just a heads up that `InfoBlock` UI will change. Maybe we could use a "simple" confirmation layout here?

![image](https://user-images.githubusercontent.com/7346473/119531257-c2046280-bd51-11eb-8d51-95addde65c1a.png)

Reply:
Oh, thanks for the heads up! I'll use a different element here